scripts/pre_commit_linter.py

```diff
@@ -714,11 +718,142 @@ def _check_spacing(all_files):
     return summary_messages
 
 
+def _check_comments(all_files):
+    """This functions checks for a period at the end of comment."""
+    print 'Starting comment checks'
```

Comment:
`... to ensure that they end in a period`

Reply:
Done!